frontend/app/mocks/db.ts

```diff
@@ -249,7 +228,7 @@ for (let i = 0; i < numberOfLetters; i++) {
 
   db.letter.create({
     userId: defaultUser.id,
-    letterType: faker.helpers.arrayElement(seededLetterTypes),
+    letterType: faker.helpers.arrayElement(['775170000', '775170001']), // @see /power-platform-data/letter-types.json
```

Comment:
Import and use the `/power-platform-data/letter-types.json` JSON instead?

```suggestion
    letterType: faker.helpers.arrayElement(letterTypes.value[0].OptionSet.Options).Value.toString(),
```

Reply:
Good idea - fixed